README.md

```diff
@@ -32,9 +32,12 @@ OR
 
 TODO: npm での使用方法を書く
 
-
 ## Usage
 
+NES.css is only providing component. You can define your layout.
+
+Default font is `Press Start 2P`. When you using this framework with any language except English (link Japanese), please use another font as necessary.
```

Comment:
Proposed:

> The default font is `Press Start 2P` which only supports English characters. When you're using this framework with any language except English, please use another font. 

Should `link Japanese` be an actual link?

Reply:
It would be good to keep a permalink alternative available for this font

Reply:
Here's the link front Dafont https://www.dafont.com/press-start-2p.font

Reply:
That's a typo. It's "like".